spec/features/consumer/shopping/checkout_spec.rb

```diff
@@ -94,6 +94,32 @@
           expect(user.reload.ship_address.address1).to eq '123 Your Head'
         end
 
+        context "when the user and customer have existing default addresses" do
+          let(:existing_address) { create(:address) }
+
+          before do
+            user.bill_address = existing_address
+            user.ship_address = existing_address
+          end
+
+          it "updates billing address and shipping address" do
+            expect(order.bill_address).to be_nil
+            expect(order.ship_address).to be_nil
+
+            place_order
+            expect(page).to have_content "Your order has been processed successfully"
+
+            expect(order.reload.bill_address.address1).to eq '123 Your Head'
+            expect(order.reload.ship_address.address1).to eq '123 Your Head'
+
+            expect(order.customer.bill_address.address1).to eq '123 Your Head'
+            expect(order.customer.ship_address.address1).to eq '123 Your Head'
+
+            expect(user.reload.bill_address.address1).to eq '123 Your Head'
+            expect(user.reload.ship_address.address1).to eq '123 Your Head'
+          endI now have tests
```

Comment:
typo here

Reply:
Removed :+1:

Reply:
I force-pushed out of habit. Sorry Luis! :see_no_evil:

Reply:
it's ok! just dont change the magic spec!